schema_test.go

```diff
@@ -517,6 +523,7 @@ func runJSONTests(t *testing.T, testFilepaths []string) {
 	tests := 0
 	passed := 0
 	for _, path := range testFilepaths {
+		// fmt.Println("\nTesting: " + path)
```

Comment:
it'd be great if these ran as subtests. it'll drop the need for this comment and give more informative test output.
```go
for _, path := range testFilepaths {
  t.Run(path, func(t *testing.T) {
    // rest of loop body...
  })
}
```